### PR TITLE
Add `UKCC8` tournament article

### PR DESCRIPTION
**Pull request by Bazzadwarf**
Almost done, not many more to do

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Bazzadwarf** [comment]
Should be good to go

File: wiki/Tournaments/UKCC/8/en.md [hunk 1]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+
+### Qualifiers
+
+See the [Qualifiers results sheet](https://docs.google.com/spreadsheets/d/1qBNQMfkLaujAMj8mQW4gTvejIhm6Wz7TOlS32IZOr64/edit#gid=174781692) for a breakdown of the qualification results.
+
+## Ruleset
+
+- This tournament is a 2v2 double-elimination British osu! tournament.
+- The structure of the tournament will be:
+  - One qualification rounds where 32 teams progress
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:561]:
```suggestion
  - One qualification round where 32 teams progress
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 2]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+  - Group stage where 16 team progress
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:562]:
```suggestion
  - Group stage where 16 teams progress
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 3]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+  - Double Elimination Bracket
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:563]:
```suggestion
  - Double-elimination bracket
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 4]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- To be able to participate, players must have an **![][flag_GB] United Kingdom or ![][flag_IE] Ireland flag or be able to provide proof of residency to the hosts.**
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:564]:
```suggestion
- To be able to participate, players must have a **![][flag_GB] United Kingdom or ![][flag_IE] Ireland flag or be able to provide proof of residency to the hosts**.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 5]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- Ranks will be taken during and when signups end. If your teams rank after signups end is under 1000, you will not be eligible to participate in the tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:565]:
```suggestion
- Ranks will be taken during and at the end of signups. If your team rank after signups end is under 1000, you will not be eligible to participate in the tournament.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 6]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- Matches will be **Team Vs**, with **ScoreV2** as the scoring system used.
+- Ranks are used to calculate seeding. Once the tournament starts, your rank is locked and will not affect your seed.
+- All staff (apart from the hosts, mappool selectors and referees) are allowed to participate in the tournament.
+- All players are to treat staff and other participants with a high level of respect. Any kind of mistreatment on the discord server or during matches (this includes homophobia, racism and toxicity) will result in a **ban** and will be **blacklisted** from participating from future UKCC tournaments.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:569]:
```suggestion
- All players are to treat staff and other participants with a high level of respect. Any kind of mistreatment on the Discord server or during matches (this includes homophobia, racism and toxicity) will result in a **ban** and the offending player will be **blacklisted** from participating from future UKCC tournaments.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 7]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- ***Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.***
+- Players are **required** to join the public Discord server so that they are contactable by other players and staff. If a player can't be contacted or is not responding to messages through Discord, in-game messages, or forum PMs, contact the staff.
+- Mappools are to be released the Monday before a stage is due to happen.
+
+### Reschedules
+
+- Schedules for the next round are released on the Monday before a stage is due to happen and **all** matches will be scheduled for the weekend. You can reschedule for weekdays at the staffs discretion.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:576]:
```suggestion
- Schedules for the next round are released on the Monday before a stage is due to happen and **all** matches will be scheduled for the weekend. You can reschedule for weekdays at the staff's discretion.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 8]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- In order to reschedule, you must contact your opponents, agree on a time and be able to provide **proof** (by form of a screenshot/image) of the agreed reschedule to the Hosts.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:577]:
```suggestion
- In order to reschedule, you must contact your opponents, agree on a time, and be able to provide **proof** (by form of a screenshot/image) of the agreed reschedule to the hosts.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 9]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- In effort to ensure as many matches take place as possible, reschedules are allowed up to **2 hour before your match is due to be played**. However, your opponent holds the right to **refuse a reschedule if asked less than 12 hours from the scheduled time**.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:578]:
```suggestion
- In effort to ensure that as many matches take place as possible, reschedules are allowed up to **2 hours before your match is due to be played**. However, your opponent holds the right to **refuse a reschedule if asked less than 12 hours from the scheduled time**.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 10]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- If it is impossible to find a time to reschedule to that suits both teams, both teams should talk to the Hosts (Bae- or Yazzehh) and we will attempt to sort something out.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:579]:
```suggestion
- If it is impossible to find a time to reschedule to that suits both teams, both teams should talk to the hosts ([Bae-](https://osu.ppy.sh/users/6576972) or [Yazzehh](https://osu.ppy.sh/users/7068973)) who will attempt to sort something out.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 11]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+- When both teams are in the lobby and ready, the referee will ask the teams to !roll. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).
+
+### Banning
+
+- Teams will have **two** bans.
+- Teams have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
+
+### Warmups
+
+- There will be no warmups in Qualification rounds. After this, warmup maps must be a submitted beatmap, and have a drain time of 4 minutes or less.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:601]:
```suggestion
- There will be no warmups in Qualification rounds. After this, warmup maps must be submitted beatmaps, and have a drain time of 4 minutes or less.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 12]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+
+### Picking
+
+- Teams have **2 minutes** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
+- For Freemod picks, teams must select any combination of the following mods: **Hidden, HardRock**.
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:606]:
```suggestion
- For FreeMod picks, teams must select any combination of **Hidden or HardRock**.
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 13]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| zyphox big noob | 0 | **5** | **Werther's Originals** | [#1](https://osu.ppy.sh/community/matches/60622403) |
+| **Maids of Neuroscience** | **5** | 0 | Goin't'MosqueBro | [#1](https://osu.ppy.sh/community/matches/60626868) |
+| **Diu Lei Lou Mo** | **5** | 1 | Degenerates buying gf | [#1](https://osu.ppy.sh/community/matches/60626908) |
+| smol players smol scores | 1 | **5** | **Karthy and AJT's Team** | [#1](https://osu.ppy.sh/community/matches/60618166) |
+| :nandos: | 4 | **5** | **Gaurdian's of Valhalla** | [#1](https://osu.ppy.sh/community/matches/60618200) |
+
+### Group Stage
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:463]:
```suggestion
### Group stage
```

File: wiki/Tournaments/UKCC/8/en.md [hunk 14]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+  - [Taku Inoue - Yoake made Ato 3 Byou (Nerova Riuz GX) [apaffy's Extra]](https://osu.ppy.sh/beatmapsets/768893#osu/1643448)
+  - [ONE OK ROCK - NO SCARED (HabiHolic) [Insane]](https://osu.ppy.sh/beatmapsets/119277#osu/306278)
+- HardRock
+  - [Arctic Monkeys - Still Take You Home (Milan-) [BOUYAAA's Insane]](https://osu.ppy.sh/beatmapsets/631466#osu/1491730)
+  - [DJ YOSHITAKA - Bloody Tears(IIDX EDITION) (Gabi) [Insane]](https://osu.ppy.sh/beatmapsets/6598#osu/29731)
+- DoubleTime
+  - [ITO KASHITARO - Boku no Hosomichi (Raose) [Journey]](https://osu.ppy.sh/beatmapsets/394691#osu/859034)
+  - [Hana - Sakura no Uta (Alyce) [Light Insane]](https://osu.ppy.sh/beatmapsets/376552#osu/824390)
+
+## Match Results
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:316]:
```suggestion
## Match results
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/UKCC/8/en.md [hunk 15]
```diff
@@ -17,4 +57,566 @@ The **UK Community Cup 8** (***UKCC8***) was a British 2v2 double-elimination os
...
+
+### Finals
+
+- NoMod
+  - [Belmont - Overstepping (Sharkie) [Extreme]](https://osu.ppy.sh/beatmapsets/665580#osu/1408543)
+  - [xi remixed by cosMo@bousouP - FREEDOM DiVE [METAL DIMENSIONS] (Cherry Blossom) [Extra]](https://osu.ppy.sh/beatmapsets/905158#osu/1907933)
+  - [Kuroneko Dungeon - Ryoushi no Umi no Lindwurm (Crystal) [Yoru's Expert]](https://osu.ppy.sh/beatmapsets/405167#osu/890915)
+  - [Endorfin. vs Feryquitous - Sincuvate (Nokris) [Nokris. vs NeilPerry]](https://osu.ppy.sh/beatmapsets/739011#osu/1559554)
+  - [Kohmi Hirose - Promise (Luscent) [geddan]](https://osu.ppy.sh/beatmapsets/1094694#osu/2287891)
+  - ["Cranky vs. MASAKI - ouroboros -twin stroke of the end- (celerih) [Flask's Expert]"](https://osu.ppy.sh/beatmapsets/1052054#osu/2226504)
```
Comment (Walavouchey) [inline wiki/Tournaments/UKCC/8/en.md:165]: ?

```suggestion
  - [Cranky vs. MASAKI - ouroboros -twin stroke of the end- (celerih) [Flask's Expert]](https://osu.ppy.sh/beatmapsets/1052054#osu/2226504)
```

Comment (Bazzadwarf) [reply wiki/Tournaments/UKCC/8/en.md:165]: 🤷

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**